File: src/data/synergyElements.ts
/** Damage elements selectable as Element synergy sub-types (006). */
export const SYNERGY_ELEMENTS = [
  "Kinetic",
  "Solar",
  "Arc",
  "Void",
  "Stasis",
  "Strand",
  "Prismatic",
] as const;

export type SynergyElement = (typeof SYNERGY_ELEMENTS)[number];

File: flutter/packages/sandbox_data/lib/src/synergy_elements.dart
// Damage elements selectable as Element synergy sub-types (product 006).
// Port of `src/data/synergyElements.ts`.

const List<String> synergyElements = [
  'Kinetic',
  'Solar',
  'Arc',
  'Void',
  'Stasis',
  'Strand',
  'Prismatic',
];

// Returns true when [name] is a known synergy element (exact wire match).
bool isSynergyElement(String name) => synergyElements.contains(name);
